howto/instance/create-instance.md

```diff
@@ -4,13 +4,13 @@ To launch an application or an image, Anbox Cloud creates an instance for it. To
 
 ## Using the dashboard
 
-Select *Create Instance* on the *Instances* view to launch an instance. The information required for launching an instance depends on whether you are creating the instance from an application or an image.
+*Create Instance* in the instances view to launch an instance. The information required for launching an instance depends on whether you are creating the instance from an application or an image.
 
 When you create the instance from an application, the attributes you define for the application decide most of the properties of the instance.
 
 When you create the instance from an image, you can define the properties of the instance during its creation.
 
-Once you *Create and Start* an instance by providing the necessary attributes, you can view the instance and its status on the *Instances* view.
+Once you *Create and Start* an instance by providing the necessary attributes, you can view the instance and its status in the instances list view.
```

Comment:
Isn't this also the case when they only "create"?

Reply:
Yes, I have generalized the term to mean the action instead of the button.